src/Components/Facility/FacilityCreate.tsx

```diff
@@ -456,19 +456,22 @@ export const FacilityCreate = (props: FacilityProps) => {
       const res = await dispatchAction(
         facilityId ? updateFacility(facilityId, data) : createFacility(data)
       );
+
       if (res && res.data) {
```

Comment:
We can merge these two `if` statements.

```
if (res && res.data) {
    if (res.status === 200) {
```

```suggestion
      if (res && res.status === 200 && res.data) {
```